catalogue/webapp/components/SearchPageLayout/SearchPageLayout.tsx

```diff
@@ -115,7 +115,30 @@ const SearchLayout: FunctionComponent<{ hasEventsExhibitions: boolean }> = ({
 
   const updateUrl = (form: HTMLFormElement) => {
     const formValues = formDataAsUrlQuery(form);
-
+    if (
+      formValues['production.dates.from'] &&
+      Array.isArray(formValues['production.dates.from']) &&
+      formValues['production.dates.to'] &&
+      Array.isArray(formValues['production.dates.to'])
+    ) {
+      let to = '',
+        from = '';
+
+      for (let i = 0; i < formValues['production.dates.from'].length; i++) {
+        const element = formValues['production.dates.from'][i];
+        if (element.trim() !== '') {
+          from = element.trim();
+        }
+      }
+      formValues['production.dates.from'] = from;
+      for (let i = 0; i < formValues['production.dates.to'].length; i++) {
+        const element = formValues['production.dates.to'][i];
+        if (element.trim() !== '') {
+          to = element.trim();
+        }
+      }
+      formValues['production.dates.to'] = to;
+    }
```

Comment:
Could we leave a comment to explain the above? At Alex's suggestion on my PR review, I created a `catalogue/webapp/utils/search.ts` file too for Sort functions, where we also tried to leave comments explain various logic, might be worth adding that function there? There's a Filters section at the bottom!

Reply:
no longer needed!
So this code is gone